### PR TITLE
Kodi keypress events

### PR DESCRIPTION
**Pull request by starkillerOG**
## Proposed change

Add documetation for Kodi keypress events, see PR: https://github.com/home-assistant/core/pull/93321

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/93321
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by emontnemery — CHANGES_REQUESTED** [review]
This needs to be updated to reflect the changes in the parent PR

**Comment by starkillerOG** [comment]
@emontnemery I made the changes needed.
However the checks fail with:
`Liquid Exception: Liquid syntax error (line 422): Expected end_of_string but found comparison in "{{trigger.event.data.data.key == "volume_up"}}" in /opt/build/repo/source/_integrations/kodi.markdown`
Do you know how to resolve that?

File: source/_integrations/kodi.markdown [hunk 1]
```diff
@@ -390,3 +390,66 @@ data:
...
+data:
+  type: keypress
+  device_id: 72e5g0ay5621f5d719qd8cydj943421a
+  entity_id: media_player.kodi_livingroom
+  sender: KodiLivingroom
+  data:
+    key: volume_up
+```
+
+A example of a automation to turn up/down the volume of a receiver using the event:
```
Comment (frenck) [inline source/_integrations/kodi.markdown:432]:
```suggestion
A example of a automation to turn up/down the volume of a receiver using the event:

{% raw %}

```

File: source/_integrations/kodi.markdown [hunk 2]
```diff
@@ -390,3 +390,66 @@ data:
...
+            target:
+              entity_id: media_player.receiver
+      - conditions:
+          - condition: template
+            value_template: "{{trigger.event.data.data.key=='volume_down'}}"
+        sequence:
+          - service: media_player.volume_down
+            target:
+              entity_id: media_player.receiver
+```
```
Comment (frenck) [inline source/_integrations/kodi.markdown:461]:
````suggestion
```

{% endraw %}
````

File: source/_integrations/kodi.markdown [hunk 3]
```diff
@@ -390,3 +390,71 @@ data:
 {% endconfiguration %}
 
 To use notifications, please see the [getting started with automation page](/getting-started/automation/).
+
+## Keypress events
+key presses of keyboards/remotes can be overwritten in Kodi and configured to send a event to Home Assistant, which can then be used in automations to for instance turn up/down the volume of a TV/receiver.
```
Comment (frenck) [inline source/_integrations/kodi.markdown]:
```suggestion

key presses of keyboards/remotes can be overwritten in Kodi and configured to send an event to Home Assistant, which can then be used in automations to, for instance, turn up/down the volume of a TV/receiver.
```

File: source/_integrations/kodi.markdown [hunk 4]
```diff
@@ -390,3 +390,71 @@ data:
...
+
+A keypress can be overwritten in Kodi by using the [Kodi keymap XML](https://kodi.wiki/view/Keymap) or from within the Kodi GUI using the [Keymap Editor add-on](https://kodi.wiki/view/Add-on:Keymap_Editor).
+
+A example of the Kodi keymap configuration using XML which will overwrite the volume_up/volume_down buttons and instead send a event to HomeAssistant:
```
Comment (frenck) [inline source/_integrations/kodi.markdown]:
```suggestion
An example of the Kodi keymap configuration using XML, which will overwrite the volume_up/volume_down buttons and instead send an event to HomeAssistant:

```

File: source/_integrations/kodi.markdown [hunk 5]
```diff
@@ -390,3 +390,71 @@ data:
...
+</keymap>
+```
+The `"KodiLivingroom"` can be set to any value and will be present in the event data as the `"sender"`
+The `"OnKeyPress"` is needed to identify the event in Home Assistant, do not change this.
+The `{"key":"volume_up"}` can contain any JSON which will be present in the event data under the `"data"` key, normally this is used to identitfy which key was pressed.
+
+For possible keyboard key names see: https://kodi.wiki/view/List_of_keynames
+For other actions see: https://kodi.wiki/view/Keymap#Keynames
+
+For the example abbove, when the volume up key is pressed, a event in Home Assistant will be fired that looks like:
```
Comment (frenck) [inline source/_integrations/kodi.markdown]:
```suggestion

The `"KodiLivingroom"` can be set to any value and will be present in the event data as the `"sender"`
The `"OnKeyPress"` is needed to identify the event in Home Assistant, do not change this.
The `{"key":"volume_up"}` can contain any JSON which will be present in the event data under the `"data"` key, normally this is used to identify which key was pressed.

For possible keyboard key names, see: https://kodi.wiki/view/List_of_keynames
For other actions, see: https://kodi.wiki/view/Keymap#Keynames

For the example above, when the volume up key is pressed, an event in Home Assistant will be fired that looks like this:
```

**Review by frenck — APPROVED** [review]
Thanks, @starkillerOG 👍 

../Frenck

**Comment by starkillerOG** [comment]
@frenck thank you very much for fixing the CLI and the tweaks!
Could not figure it out.

**Comment by frenck** [comment]
No problem at all 👍

**Pull request merged**